KEEP/pkg_skel/go

```diff
@@ -1,12 +1,19 @@
+[mirrors]
+
+[vars]
+filesize=
+sha512=
+pkgver=1
+
 [deps]
 go
-git
 
 [build]
-repo="github.com/user/repo"
+baseurl=github.com/example
+repo=name
 
-dest="$butch_install_dir""$butch_prefix"
-srcdest="$S"/build/"$butch_package_name"
-GOROOT="$butch_root_dir"/opt/go GOPATH="$dest" go get "$repo"
-[ -d "$srcdest" ] && rm -rf "$srcdest"
-mv -f "$dest"/src "$srcdest"
+mkdir -p butch_build_tmp/src/"$baseurl"
+ln -sf "$butch_build_dir" butch_build_tmp/src/"$baseurl"/"$repo"
+GOPATH="$butch_build_dir"/butch_build_tmp \
+  GOBIN="$butch_install_dir""$butch_prefix"/bin \
```

Comment:
i wonder about the usage of "$butch_install_dir""$butch_prefix" here.
is it because go insists of being installed into /opt/go (which is really idiotic and interferes with our packet management) ? if so, we need to hardcode /opt/go, unless we find a workaround... for example since /opt/go/bin/* is linked to /bin/*, could we set GOBIN to $butch_root_dir/bin ?

Reply:
go insists in installing into GOPATH/bin, in this case being "$butch_build_dir"/butch_build_tmp/bin/ .
GOROOT is /opt/go which points to where go itself is installed and is usually not the same as the GOPATH. I put "$butch_install_dir""$butch_prefix" because that is the usual value for $dest.

Reply:
I don't see anything wrong with the way it is now.

Reply:
oh right. seems i have mistaken GOROOT with GOBIN